src/test/java/com/conveyal/gtfs/GTFSFeedTest.java

```diff
@@ -0,0 +1,27 @@
+package com.conveyal.gtfs;
+
+import org.junit.Test;
+
+import java.io.File;
+
+import static com.conveyal.gtfs.TestUtils.getResourceFileName;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class GTFSFeedTest {
+    @Test
+    public void canDoRoundtripLoadAndWriteToZipFile() {
+        String outZip = getResourceFileName("fake-agency-output.zip");
+        File file = new File(outZip);
+        file.delete();
+        try {
+            GTFSFeed feed = new GTFSFeed();
+            feed.fromFile(getResourceFileName("fake-agency.zip"));
+            feed.toFile(outZip);
+            assertThat(file.exists(), equalTo(true));
+            // TODO: assert that rows of data were written
```

Comment:
Ideally we'd then read it back in and do some comparison. There should already be tests that do this for the MapDB backed code.

Reply:
I added some tests for this, but they are failing because data isn't being written.  I'm not sure if I'm using the feed instance correctly here.